backend/tests/test_dynamic_routes.py

```diff
@@ -0,0 +1,600 @@
+
+        response = client.get('/person?age.gt=0&age.lt=20&offset=2')
+        assert response.json()['entities'] == []
+
+    @pytest.mark.parametrize(['q', 'resp'], [
+        ('age.gt=0&age.lt=20',                            ['jane']),
+        ('offset=1&age.gt=0&age.lt=20',                   []),
+        ('all=True&age.gt=0&age.lt=20',                   ['jack_del', 'jane']),
+        ('deleted_only=True&age.gt=0&age.lt=20',          ['jack_del']),
+        ('deleted_only=true&offset=1&age.gt=0&age.lt=20', []),
+    ])
+    def test_get_with_filters_and_deleted(self, dbsession, client, request, q, resp):
+        dbsession.execute(update(Entity).where(Entity.slug == 'Jack').values(deleted=True))
+        dbsession.commit()
+        resp = [request.getfixturevalue(i) for i in resp]
+        response = client.get(f'/person?{q}')
+        assert response.json()['entities'] == resp
+
+    def test_ignore_invalid_filter(self, dbsession, client, jack, jane):
+        response = client.get('/person?age.gt=0&age.lt=20&qwe.rty=123')
+        assert response.json()['entities'] == [jack, jane]
+
+        response = client.get('/person?age.gt=0&age.lt=20&friends.lt=1234')
+        assert response.json()['entities'] == [jack, jane]
+
+        response = client.get('/person?age.qwe=1234')
+        assert response.json()['entities'] == [jack, jane]
+
+    def test_ignore_filters_for_list_and_fk(self, dbsession, client, jack, jane):
+        response = client.get('/person?friends=1')
+        assert response.json()['entities'] == [jack, jane]
+
+class TestRouteCreateEntity:
+    def test_create(self, dbsession, client):
+        p1 = {
+            'name': 'mike',
+            'slug': 'Mike',
+            'nickname': 'mike',
+            'age': 10,
+            'friends': [],
+        }
+        response = client.post(f'/person', json=p1)
+        assert response.json() == {'id': 3, 'slug': 'Mike', 'name': 'mike', 'deleted': False}
+        
+        mike = dbsession.execute(select(Entity).where(Entity.id == 3)).scalar()
+        assert mike.get('nickname', dbsession).value == 'mike'
+        assert mike.get('age', dbsession).value == 10
+        assert mike.get('friends', dbsession) == []
+
+        p2 = {
+            'name': 'john',
+            'slug': 'John',
+            'nickname': 'john',
+            'age': 10,
+            'friends': [3, 1],
+            'born': '2021-10-20T13:52:17',
+        }
+        response = client.post(f'/person', json=p2)
+        assert response.json() == {'id': 4, 'slug': 'John', 'name': 'john', 'deleted': False}
+
+        john = dbsession.execute(select(Entity).where(Entity.id == 4)).scalar()
+        assert john.get('nickname', dbsession).value == 'john'
+        assert john.get('age', dbsession).value == 10
+        assert [i.value for i in john.get('friends', dbsession)] == [3, 1]
+        assert john.get('born', dbsession).value == datetime(2021, 10, 20, 13, 52, 17)
```

Comment:
I had not noticed before, or I was just assuming, but timestamps without timezone might become confusing.

If it's not too much effort, I'd appreciate a change to use timezones (and set a default timezone, if none is provided). But we could address this in a separate issue/PR.

Reply:
Ok, so, I changed `value = Column(DateTime)` to `value = Column(DateTime(timezone=True))` in `ValueDatetime` and now PG saves date in column of type `timestamp with time zone` and converts all datetimes to UTC. Will that work for you?

Reply:
That will work... If we want another default timezone (i.e. one that differs from the default timezone of the DB) in the future, we could address that in the field validation, right?

Reply:
> If we want another default timezone

When doing what? Getting/passing dates from/to backend? I didn't really get the question🤔

Reply:
If your question is "Is it possible to return dt in timezone other than defined in DB by using pydantic validators?" then yes, I guess so

Reply:
I was just thinking out loud: If I understood you correctly, if a timestamp does not define the timezone, the database uses it's default timezone. In a worst case scenario, where we cannot change the timezone of the database, I was thinking about adding a default timezone as part of the value validation.

But I think, the timezone configured in the database should be sufficient.

Reply:
> If your question is

Yes, that was the idea. But I don't think we need to implement this now.